### DISCUSSION TITLE
Call churn, but take my money

### DISCUSSION THREAD

**Discussion by eknowlton** [tteck/Proxmox#470] [General]
Awesome repo, very helpful. Sorry if this just wastes your time but there is no "buy me a coffee" button.

I planned on installing a lot of these services and you've just made it very easy.

**Comment by tteck** [comment]
Thanks for the kind words! Just the fact that people find my work useful is payment enough.

I'll test the waters
[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/D1D7EP4GF)

**Reply by dimatx** [reply]
Agreed with @eknowlton. This is a very valuable repo and has saved me many hours and has given me an opportunity to roll out services that I wouldn't have had the time to tinker with otherwise.

**Reply by mschabhuettl** [reply]
Thanks for your work! Saved me tons of hours. Bought you a Coffee! ❤️

**Reply by Deekerman** [reply]
10/10 would support again. Magic Scrips are Magic! (at least to my barbarian self)

**Reply by AlexGeek77** [reply]
I’ve been a fan for a while. I was doing this on my own before but your install scripts are far more valuable and consistent.
